### ISSUE TITLE
Some minor concerns about the training hyper parameters.

### ISSUE THREAD

**Issue by Guzaiwang** [Junda24/MonSter#10]
First of all, thanks for your excellent mono foundation model-based stereo matching.

In the paper, you mention that "one-cycle learning rate schedule with a learning rate of 2e-4 and train MonSter with a batch size of 8 for 200k steps as the pretrained model." 
However, in the training_sceneflow.yaml config file, the parameters appear to be different.
Could you please clarify which training parameters I should follow to train from scratch?

**Comment by Junda24** [comment]
> First of all, thanks for your excellent mono foundation model-based stereo matching.
> 
> In the paper, you mention that "one-cycle learning rate schedule with a learning rate of 2e-4 and train MonSter with a batch size of 8 for 200k steps as the pretrained model." However, in the training_sceneflow.yaml config file, the parameters appear to be different. Could you please clarify which training parameters I should follow to train from scratch?

Sorry for confusion. The parameter 1e - 4 is for when performing finetuning on the mixed data. I have made the modification.

**Comment by Guzaiwang** [comment]
thanks!
